docs/DeveloperGuide.adoc

```diff
@@ -321,9 +321,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |courier |know customers' payment methods |be prepared to collect any payment upon delivery
 
-|=======================================================================
+|`* *` |courier |know the nearest popstation/pick-up location for returned parcels |plan my route to pick up parcels to be returned
 
-_{More to be added}_
+|`* *` |advanced courier |use shorter versions of a command |type a command faster
+
+|`* *` |courier |keep track of the amount I have received for the day’s orders and the change I should give back |know whether the cash balance is correct at the end of the day
+
+|`* *` |caring courier |generate CSV based on what order I select |send the list of orders to my colleagues
+
+|`* *` |courier |report areas of traffic congestion to my colleagues |help my colleagues reduce their delivery times. (Requires Internet Connection)
```

Comment:
Since the app does not use the internet, maybe this feature should belong in the unlikely to have category.